app/assets/javascripts/jquery-mentions-input/jquery.mentionsInput.js

```diff
@@ -470,7 +470,7 @@
     //Resets the text area
     function resetInput(currentVal) {
       mentionsCollection = [];
-      var mentionText = utils.htmlEncode(currentVal);
+      var mentionText = currentVal;
```

Comment:
@ben @grosser I removed the need to use the Slack tags by enabling full parsing: https://github.com/zendesk/samson/pull/1208

We probably should reinstate this as it is a security issue.